extensions/resteasy/runtime/src/main/java/io/quarkus/resteasy/runtime/standalone/VertxRequestHandler.java

```diff
@@ -97,7 +97,16 @@ private void dispatch(RoutingContext routingContext, InputStream is, VertxOutput
             VertxHttpResponse vertxResponse = new VertxHttpResponse(request, dispatcher.getProviderFactory(),
                     request.method(), allocator, output);
             // client address may not be available with VirtualHttp
-            SocketAddress socketAddress = request.remoteAddress();
+            SocketAddress socketAddress = null;
+            try {
+                socketAddress = request.remoteAddress();
+            } catch (Exception e) {
```

Comment:
catching `Exception` is probably overly broad.  If we know that vertx is throwing an NPE why don't we catch that here?

Reply:
@evanchooly, that sounds fine to me. I've now updated this PR to just catch `NullPointerException`.